fishtank/src/cluster.ts

```diff
@@ -1,11 +1,19 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-import { Docker } from './backend'
+import { Docker, Labels } from './backend'
 import { Node } from './node'
 
 export const DEFAULT_IMAGE = 'ironfish:latest'
+export const DEFAULT_BOOTSTRAP_NODE_NAME = 'bootstrap'
 export const CLUSTER_LABEL = 'fishtank.cluster'
+export const NODE_ROLE_LABEL = 'fishtank.node.role'
+export const BOOTSTRAP_NODE_ROLE = 'bootstrap'
+
+export type BootstrapOptions = {
```

Comment:
This options are the same for spawn(), it's the container options: name and image. Can you make it generic and use it for spawn too?

Reply:
These are the same for now, but they will deviate in the future as they serve two different purposes. spawn() is for spinning up new nodes, bootstrap() is for setting up the network and initializing the chain.

Reply:
These options are the container config, which serves both bootstrap container and node container.  If in the future, there are cases they need to be separate, you can do it.  For this PR and the first milestone, please make it generic and  serve both.

Reply:
If we use the same type for things that are not semantically equivalent, then moving to a different type later on would be a breaking change. Also, if I merge these two types now, I will have to split them again in my next PR. The "future" is "this week" in this case. I can merge the types to unblock things, but I really do not see a reason why.

Reply:
```
export type ContainerOptions = {
  nodeName?: string
  nodeImage?: string
}
```
```
async bootstrap(options?: ContainerOptions): Promise<void> {}
async spawn(options: { container?: ContainerOptions,  config?: Partial<ConfigOptions>}): Promise<Node> {
```
This is the interface I am thinking, what field you need to add in your next PR that needs to split them?